README.md

```diff
@@ -45,8 +45,9 @@ GitHub Action for automatically syncing LeetCode submissions to a GitHub reposit
 
         steps:
         - name: Sync
-          uses: joshcai/leetcode-sync@v1.3
+          uses: joshcai/leetcode-sync@v1.4
```

Comment:
Can you revert this? I'll need to create a new release with these changes before this is valid